examples/facerec_from_webcam_faster.py

```diff
@@ -45,14 +45,14 @@
     # Only process every other frame of video to save time
     if process_this_frame:
         # Resize frame of video to 1/4 size for faster face recognition processing
-        small_frame = cv2.resize(frame, (0, 0), fx=0.25, fy=0.25)
+        small_frame = cv2.resize(frame, (0, 0), fx=1, fy=1)
 
         # Convert the image from BGR color (which OpenCV uses) to RGB color (which face_recognition uses)
-        rgb_small_frame = small_frame[:, :, ::-1]
-        
+        rgb_frame = cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)
```

Comment:
It would work better if `rgb_frame = cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)` is replaced with `rgb_frame = cv2.cvtColor(small_frame, cv2.COLOR_BGR2RGB)`

Reply:
This fix is for faster, the issue which defined this fix gave the solution for https://github.com/ageitgey/face_recognition/blob/master/examples/facerec_from_webcam.py